compiler/damlc/daml-doc/src/DA/Daml/Doc/HaddockParse.hs

```diff
@@ -67,16 +67,23 @@ mkDocs opts fp = do
                     case stripInstanceSuffix (cl_name classDoc) of
                         Nothing -> Left classDoc
                         Just templateName -> Right (templateName, classDoc)
-            templateInstanceMap = MS.fromList templateInstanceClasses
+            templateInstanceClassMap = MS.fromList templateInstanceClasses
 
             md_name = dc_modname
             md_anchor = Just (moduleAnchor md_name)
             md_descr = modDoc dc_tcmod
-            md_templates = getTemplateDocs ctx typeMap templateInstanceMap
+            md_templates = getTemplateDocs ctx typeMap templateInstanceClassMap
             md_functions = mapMaybe (getFctDocs ctx) dc_decls
-            md_adts
+
+            filteredAdts -- all ADT docs without templates or choices
```

Comment:
`without`? This looks more like it only has `dc_templates` and `dc_choices`.

Reply:
It is without :-) that's what `withoutKeys` does

Reply:
Apparently I can’t read :slightly_smiling_face: